src/main/java/uk/gov/hmcts/darts/stub/services/download/CsvDownloadFactory.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.darts.stub.services.download;
+
+
+import java.io.IOException;
+import java.util.List;
+
+
+public class CsvDownloadFactory {
+
+    private CsvDownloadFactory() {
+    }
+
+    public static DownloadCsvRecord getBasicCsvDownloadRecord(String eodId) throws IOException {
+        List<DownloadCsvRecord> downloadCsvRecord = CsvUtil.toDownloadCsvRecord(CsvUtil.getExpectedData());
```

Comment:
Could cache this once then reuse it each time you need it as reading from files can be an expensive operation

Reply:
done